guardian/service-catalogue
# refactor: replace cron with systemd timer

## What does this change? And why?
This replaces the daily cron with a systemd timer because: 
- Use of systemd is the [recommended](https://github.com/guardian/recommendations/blob/main/scheduled-jobs.md) way to run cron tasks
- It pushes logs into journald, which allows us to utilise [devx-logs](https://github.com/guardian/devx-logs) to ship these to Central ELK (we'll do this in a separate PR)

## How has it been verified?
- Deploy the cloudquery service
- Observe logs in journald (`journalctl -u cloudquery`)
